migration-from-4x.md

```diff
@@ -171,7 +169,7 @@ The Modern theme is no longer supported in TinyMCE 5.0.  The modern theme's UI l
 
 ## Mobile support
 
-In TinyMCE 4.x, mobile support was introduced bundled with a new theme and configuration settings. TinyMCE 5.0 makes this process seamless where mobile support comes out of the box without any additional configurations.  The new theme is now responsive on tablets, where the classic desktop theme will be displayed, and on smaller devices like phones. It now shows a UI responsive to touchscreens. TinyMCE 5.0 mobile will be an exciting space to watch.
+In TinyMCE 4.x, mobile support was introduced bundled with a new theme and configuration settings. TinyMCE 5.0 makes this process seamless where mobile support comes out of the box without any additional configurations. The mobile theme will be automatically loaded if a user views the editor on a mobile phone. TinyMCE 5.0 mobile will be an exciting space to watch.
```

Comment:
Perhaps should say something about existing configurations will still work as well, though we need to re-add that configuration section I linked you privately @shikhanansi then link that here

Reply:
Understood.